### DISCUSSION TITLE
​

### DISCUSSION THREAD

**Discussion by abigailwillow** [victornpb/undiscord#306] [3. Ideas]

**Comment by neontealtree** [comment]
Imho you should pick the issues you want to bring in, then reply to them directing the person who created it to this repository. Also, if you find this is manageable for yourself, then add the kofi link so people can "buy you a coffee".

**Reply by Toby222** [reply]
You can reference issues by name [in the commit message](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue). e.g. `resolves victornpb/deleteDiscordMessages#{issue number}`.

Pull Requests can be merged like normal branches, you just wouldn't get them logged as an item on your own Repo. The original PR would link back to the commit if you add the above to the merge commit message, though, unless I'm severely mistaken.

**Reply by Void48** [reply]
just wanna say thanks for keeping this alive @abbydiode

**Comment by abigailwillow** [comment]


**Reply by victornpb** [reply]
Yes please, as you already know it has been quite a challenge keeping up with this project, and I've been very busy with life. Even tho this code base is very simple, testing stuff is very time consuming.

I have to setup at least 2 dummy accounts, populate it, and then test it. I've been thinking about implementing automated tests, but even that the setup for this is quite unconventional. But it would speed up merging contributions.

Is the top of your fork currently stable (tested)?

**Reply by victornpb** [reply]
I like how you reworded a few of the documentations, I was able to reuse most of your changes, I'm still not sure if I'm going to keep the docs inside the repo or the wiki, for now I'm going to leave it enabled, as it makes editing convenient.

I think it will take me a few days to reserve the time to check in your pr. I spend most of my free time today ~2-3h reviewing the easy stuff first. Closed around 40 ish issues. And pushed a new tag with the recent ~10 PRs merged. Looks ok at first glance.

**Reply by victornpb** [reply]
I wish I could put the issues tab on read only instead of disabling it completely. I think the discussion tab works better for the general public (non developers). But it still some ideas there I want to keep track.

**Reply by victornpb** [reply]
> I think they both serve different purposes. I personally like using the issues tab and discussions separately. Not having the issues tab at all is a bit odd in my opinion.

Totally agree they serve different purposes, issues makes bug reporting for example easier on my side, only if people actually used the templates correctly. Disabling it completely was a measure out of desperation 😅.

On the plus side I saw an increase in the community answering themselves, rather than me closing issues daily.

So I rather sacrifice the issues if that means less backlog.

I just wish people would use discussion for questions and less technical more open ended stuff

And issues for actual bug reports

**Reply by victornpb** [reply]
I edited the readme links and the issue templates with uppercase letters, let's see how it turns out
